tls/tls-full/client-accounting.conf

```diff
@@ -0,0 +1,15 @@
+[req]
+default_bits = 4096
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
+[dn]
+C = US
+ST = WA
+O = Contoso, Inc.
+CN = accounting.cluster-x.hq.contoso.com
```

Comment:
[Nit] example cname doesn't really make it clear that this is a client certificate. Should we consider tweaking the name?